django/core/management/commands/makemigrations.py

```diff
@@ -188,7 +193,7 @@ def write_migration_files(self, changes):
                 self.stdout.write(self.style.MIGRATE_HEADING("Migrations for '%s':" % app_label) + "\n")
             for migration in app_migrations:
                 # Describe the migration
-                writer = MigrationWriter(migration)
+                writer = MigrationWriter(migration, self.no_header)
```

Comment:
It feels wrong to pass a negated option to `MigrationWriter`. It feels like `include_header=self.include_header`